lib/tracker.rb

```diff
@@ -0,0 +1,20 @@
+class Tracker 
+  
+  attr_reader :initial_balance, :account
+
+  def initialize(account = Account.new([]), initial_balance = 0)
+    @account = account
+    @initial_balance = initial_balance
+  end 
+
+  def calculate_balance(account, entry)
+    current_balance = @initial_balance
```

Comment:
Really nice way of calculating the balance independent of iterating through the transactions!